scalableminds/webknossos
# Continuous brush strokes

This PR ensures that brush strokes are continuous by drawing a rectangle between two circles when necessary.

### URL of deployed dev instance (used for testing):
- https://continuousbrushstrokes.webknossos.xyz

### Steps to test:
- open a volume tracing
- select the brush tool
- brush fast
- there should be no gaps in your stroke

### Issues:
- fixes #4785 

------
